services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -69,7 +71,7 @@ export class HookCallbackHandlerService {
       await this.workflowService.copyFileAndCreate(
         { documents: documents } as DefaultContextSchema,
         context.entity.id,
-        proejctIds,
+        proejctIds?.[0] as TProjectId,
```

Comment:
should use projectId (and receive ) projectId and not extract it from projectIds scope